Makefile.ci.toml

```diff
@@ -196,28 +197,45 @@ script = """
 
 [tasks.start-tikv]
 category = "CI - SERVICES"
+env = { SURREAL_LINK="https://github.com/surrealdb/surrealdb/releases/download/v1.2.0/surreal-v1.2.0.linux-amd64.tgz" }
```

Comment:
Should this be tied to 1.2.0 specifically?

Reply:
I just picked the newest. It can be any binary of surrealdb. I made it a ENV so we can change it easily.